rest/pulsar_job.go

```diff
@@ -0,0 +1,164 @@
+package rest
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"gopkg.in/ns1/ns1-go.v2/rest/model/pulsar"
+)
+
+// JobsService handles 'pulsar/apps/APPID/jobs/JOBID' endpoint.
+type PulsarJobsService service
+
+// List takes an Application ID and returns all Jobs inside said Application.
+//
+// NS1 API docs: https://ns1.com/api/#getlist-jobs-within-an-app
+func (s *PulsarJobsService) List(appId string) ([]*pulsar.PulsarJob, *http.Response, error) {
+	path := fmt.Sprintf("pulsar/apps/%s/jobs", appId)
+	req, err := s.client.NewRequest("GET", path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	jl := []*pulsar.PulsarJob{}
+	var resp *http.Response
+	resp, err = s.client.Do(req, &jl)
+	if err != nil {
+		switch errorType := err.(type) {
+		case *Error:
+			if errorType.Message == "pulsar app not found" {
+				return nil, resp, ErrAppMissing
+			}
```

Comment:
noted this question on the other pulsar PR as well, is there not a 404 status code or something to check instead of a hard coded string?